src/borg/key.py

```diff
@@ -258,8 +254,8 @@ def id_hash(self, data):
         return sha256(data).digest()
 
     def encrypt(self, chunk):
-        chunk = self.compress(chunk)
```

Comment:
just consider chunks being tuple(meta, data) infrastructure we do not want to throw away.
i introduced that concept a while ago for multiple reasons and i don't want to redo that.